person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    age > 18
```

Comment:
- [ ] Kindly fix this condition to be true if age is greater or equal to 18 as in the requirements.

Reply:
- Please fix the condition, according to the project requirements, it should check if the age is greater or equal to `18`